browsers/pools.mdx

```diff
@@ -0,0 +1,303 @@
+
+console.log(pool.available_count);
+console.log(pool.acquired_count);
+```
+
+```python Python
+pool = kernel.browser_pools.retrieve("my-pool")
+
+print(pool.available_count)
+print(pool.acquired_count)
+```
+</CodeGroup>
+
+## List pools
+
+List all browser pools in your organization.
+
+<CodeGroup>
+```typescript Typescript/Javascript
+const pools = await kernel.browserPools.list();
+
+for (const pool of pools) {
+  console.log(pool.name, pool.available_count);
+}
+```
+
+```python Python
+pools = kernel.browser_pools.list()
+
+for pool in pools:
+    print(pool.name, pool.available_count)
+```
+</CodeGroup>
+
+## Delete a pool
+
+Delete a browser pool and all browsers in it. By default, deletion is blocked if browsers are currently acquired. Use `force: true` to terminate acquired browsers and force deletion.
+
+<CodeGroup>
+```typescript Typescript/Javascript
+// Delete a pool (fails if browsers are acquired)
+await kernel.browserPools.delete("my-pool");
+
+// Force delete even if browsers are acquired
+await kernel.browserPools.delete("my-pool", { force: true });
+```
+
+```python Python
+# Delete a pool (fails if browsers are acquired)
+kernel.browser_pools.delete("my-pool")
+
+# Force delete even if browsers are acquired
+kernel.browser_pools.delete("my-pool", force=True)
+```
+</CodeGroup>
+
+## Full example
+
+This example assumes you've already created a pool named "my-pool". In practice, you'd create pools once (via the SDK, CLI, or dashboard) and then acquire from them repeatedly.
+
+<CodeGroup>
+```typescript Typescript/Javascript
+import Kernel from '@onkernel/sdk';
+import { chromium } from 'playwright';
+
+const kernel = new Kernel();
+
+// Acquire a browser from an existing pool
+const kernelBrowser = await kernel.browserPools.acquire("my-pool", {});
+
+try {
+  // Connect via CDP
+  const browser = await chromium.connectOverCDP(kernelBrowser.cdp_ws_url);
+  const context = browser.contexts()[0];
+  const page = context.pages()[0];
+
+  await page.goto('https://example.com');
+  const title = await page.title();
+  console.log(title);
+
+  await browser.close();
+} finally {
+  // Release back to pool for reuse
+  await kernel.browserPools.release("my-pool", {
+    session_id: kernelBrowser.session_id,
+  });
+}
+```
+
+```python Python
+import asyncio
+from kernel import Kernel
+from playwright.async_api import async_playwright
+
+kernel = Kernel()
+
+async def main():
+    # Acquire a browser from an existing pool
+    kernel_browser = kernel.browser_pools.acquire("my-pool")
+
+    async with async_playwright() as playwright:
+        try:
+            # Connect via CDP
+            browser = await playwright.chromium.connect_over_cdp(kernel_browser.cdp_ws_url)
+            context = browser.contexts[0]
+            page = context.pages[0]
+
+            await page.goto('https://example.com')
+            title = await page.title()
+            print(title)
+
+            await browser.close()
```

Comment:
Please remove this line. We have seen users get confused and think this destroys the Kernel browser too. Thx!

Reply:
oof bad miss on my part. sorry for the slop and ty!

Reply:
https://github.com/onkernel/docs/pull/130/commits/6b8bdd67aea081631f3d6ede9e6c6f1b4ae95933